util/dvsim/FlowCfg.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
         self.job_prefix = args.job_prefix
         self.gui = args.gui
 
+        self.interactive = args.interactive
```

Comment:
There are some edge cases handled in this source code below with `args.gui`:
https://github.com/lowRISC/opentitan/blob/2fc3a7c29dfa4d5e5942cdd3c67ea441f4db086e/util/dvsim/FlowCfg.py#L372

Could you please add `args.interactive` to it as well?